src/views/activity/Activity.vue

```diff
@@ -200,11 +209,19 @@ export default {
   },
 
   mounted: async function () {
-    this.$store.dispatch('settings/load');
+    this.$store.dispatch('views/load');
+    this.$store.dispatch('categories/load');
     await this.refresh();
   },
 
   methods: {
+    addView: function () {
+      // TODO: Open modal to ask for options like id, and name
```

Comment:
I think that naming the view is a critical feature.
Seems very strange that the user has to remember what kind of visualizations is in an specific ID of a view.